cl_manycore/libraries/bsg_manycore_tile.cpp

```diff
@@ -81,102 +82,408 @@ int hb_mc_tile_set_origin(hb_mc_manycore_t *mc, const hb_mc_coordinate_t *tile,
 	return rc;
 }
 
+
```

Comment:
Was this additional whitespace necessary?

Reply:
guess not! fixed!